meterfs/file_phx.c

```diff
@@ -0,0 +1,225 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Meterfs test file abstraction
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Aleksander Kaminski, Andrzej Glowinski, Tomasz Korniluk
+ *
+ * %LICENSE%
+ */
+
+#include <stdio.h>
+#include <sys/msg.h>
+#include <string.h>
+#include <unistd.h>
+#include <meterfs.h>
+
+#include "file.h"
+#include "unity_fixture.h"
+
+static oid_t meterfs;
+
+static inline void file_prepareDevCtl(msg_t* msg)
+{
+	msg->type = mtDevCtl;
+	msg->i.data = NULL;
+	msg->i.size = 0;
+	msg->o.data = NULL;
+	msg->o.size = 0;
+}
+
+
+int file_lookup(const char *name)
+{
+	msg_t msg;
```

Comment:
Declare just `oid_t`. `msg_t` is a huge `struct` and you need only small part of it.

Reply:
Done.